mesa/examples/basic/schelling/model.py

```diff
@@ -9,11 +9,11 @@ class Schelling(Model):
 
     def __init__(
         self,
-        height: int = 40,
-        width: int = 40,
+        height: int = 20,
+        width: int = 20,
```

Comment:
I'm curious, what's the motivation for making the grid smaller?

Reply:
@EwoutH It was just to fix the problem I mentioned above. The problem was the grid size was changing from 40 by 40 to 20 by 20 after pressing RESET button. Before reset, grid size is what is provided by default in the definition of Schelling model(40 by 40), and after pressing RESET , grid size is what is passed into SolaraViz class for making visualization ( 20 by 20 ).  
So changing grid size from 40 by 40 to 20 by 20 inside the model makes grid size uniform. However, 40 by 40 grid size could also be passed inside SolaraViz class instead of 20 by 20.